src/main/webapp/resources/js/pages/analysis/components/AnalysisSteps.jsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * This file displays the steps of the analysis
+ * (Queued, Preparing, Submitting, Running,
+ * Completing, Completed)
+ */
+
+/*
+ * The following import statements makes available all the elements
+ * required by the component
+ */
+import React, { useContext } from "react";
+import { Steps, Icon } from "antd";
+import { AnalysisContext, stateMap } from "../../../contexts/AnalysisContext";
+import { getI18N } from "../../../utilities/i18n-utilties";
+import { SPACE_MD } from "../../../styles/spacing";
+
+const Step = Steps.Step;
+
+export function AnalysisSteps() {
+  const { analysisContext } = useContext(AnalysisContext);
+  return (
+    <>
+      <Steps
+        current={stateMap[analysisContext.analysisState]}
+        status={analysisContext.isError ? "error" : "finish"}
+        style={{ paddingBottom: SPACE_MD }}
+      >
+        <Step
+          title={getI18N("analysis.state.NEW")}
+          icon={
+            analysisContext.analysisState == "NEW" ? (
```

Comment:
Stop using `==`

Reply:
Will do :) 

Fixed in 9c4e640d5c452bf1455f9b93a1b1499a418283e6